mri/generators/__init__.py

```diff
@@ -0,0 +1,6 @@
+from .base import KspaceGeneratorBase
```

Comment:
Do we have to import the Base class?

Reply:
We could do it without it, but there is a lot in common to generators. Currently only cartesian one has been develop, but extending to non-cartesian by inheriting the Base class will be  easier.